.github/workflows/pr-gpu.yaml

```diff
@@ -26,6 +26,11 @@ jobs:
             markers: 'not daily and not remote and gpu and (vision or not vision) and (doctest or not doctest)'
             pytest_command: 'coverage run -m pytest'
             composer_package_name: 'mosaicml'
+          - name: 'gpu-3.10-2.1-pytorch-nightly'
```

Comment:
Let's skip this for now... we don't really have GPUs lying around

Reply:
removed

Reply:
removed